netbox/core/utils.py

```diff
@@ -0,0 +1,26 @@
+from netbox.registry import registry
```

Comment:
should this be in netbox/core/utils.py or netbox/netbox/utils.py?

Reply:
I don't have a strong opinion, but I put it here because DataBackend lives in `core`. We should probably have a higher-level discussion about the organization of utility functions in general. I'm okay leaving this as-is for now and potentially moving prior to the v3.7 beta release.